lib/components/ScaleLegend.vue

```diff
@@ -50,12 +50,15 @@ export default {
   },
   data () {
     return {
-      cursorWrapperOffset: 0
+      cursorWrapperOffset: 0,
+      mounted: false
     }
   },
   async mounted () {
     await this.$nextTick()
     this.setCursorWrapperOffset()
+    this.setColorScaleCanvas()
```

Comment:
The return statement is not used. It might be because of the refactor, moving the method from "computed" to "methods"

Reply:
The return statement is not needed indeed. Removing it.

Reply:
https://github.com/ICIJ/murmur/pull/62/commits/c36bd8ea69fed381c11dc5932d4cc57c95013226